docker-compose.yaml

```diff
@@ -352,7 +353,7 @@ services:
       - AUTH_REALM=zaakafhandelcomponent
       - AUTH_RESOURCE=zaakafhandelcomponent
       - AUTH_SECRET=keycloakZaakafhandelcomponentClientSecret
-      - AUTH_SERVER=http://host.docker.internal:8081
+      - AUTH_SERVER=http://keycloak:8080
```

Comment:
Dit is dus de URL die extern - dus de fe - gebruikt wordt, en hostname-url is intern - dus binnen k8s netwerk - in gebruik?

Reply:
Nee precies andersom. :-) Nogal verwarrend. Zie: https://www.keycloak.org/server/hostname

Dus binnen Docker Compose van ZAC naar Keycloak gebruik je de container naam ('keycloak'). Maar via de browser (als je vanaf localhost:8080 op ZAC komt) ga je naar Keycloak via http://localhost:8081